docker-app/qfieldcloud/core/admin.py

```diff
@@ -360,6 +360,29 @@ def has_change_permission(self, request, obj):
         return False
 
 
+@admin.register(OrganizationMember)
+class OrganizationMemberAdmin(admin.ModelAdmin):
```

Comment:
We can discuss whether we require this new admin. IMO we cacn live without it and the adjustments were needed only for `MemberOrganizationInline`.

Reply:
Not needed currently